README.rst

```diff
@@ -66,10 +66,30 @@ pkgdb2::
 
     (my-pkgdb2-env)$ python createdb.py
 
+Setting up PostgreSQL
+=====================
+You can set up PostgreSQL instead of sqlite
```

Comment:
Maybe we could precise that a) it's not mandatory and b) why would one want to do this? :)